Library/Formula/cfdg.rb

```diff
@@ -0,0 +1,34 @@
+class Cfdg < Formula
+  homepage 'http://www.contextfreeart.org/'
+  url 'http://www.contextfreeart.org/download/ContextFreeSource3.0.8.tgz'
+  sha1 'bbd90c74fd0e66cb071c342da52ad016e023da2b'
+
+  depends_on 'bison' => :build
+  depends_on 'libpng'
+
+  def install
+    system 'make'
+    bin.install 'cfdg'
```

Comment:
I investigated that before writing this formula, but there's no rule to install in Makefile.